charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -183,6 +187,13 @@ spec:
             key: password
     {{- end }}
     {{- end }}
+    {{- if or .Values.broker.aop.enabled .Values.components.aop }}
```

Comment:
It's a bit confusing to have multiple flags to enable a feature...

Reply:
honestly yes, but I'm not sure why we have 2 different flags, I assume user will expect kop/aop enabled if we set either of them so just check both here.